enterprise/README.md

```diff
@@ -1,33 +1,18 @@
-# Official Flyway Teams and Enterprise Docker images
+# Official Flyway Community, Teams and Enterprise Docker images
 
-[![Docker Auto Build](https://img.shields.io/docker/cloud/automated/redgate/flyway)][docker]
+This is the official repository for [Flyway Command-line](https://documentation.red-gate.com/fd/welcome-to-flyway-184127914.html) images.
 
-[docker]: https://hub.docker.com/r/redgate/flyway/
```

Comment:
This line was kept in the OSS page

Reply:
I know - but I have no idea what it does. I suppose the safe thing is to keep it for now

Reply:
Reverted it back

Reply:
Actually it is outdated cruft (it's a badge from when we were using dockerhub to build the images - not doing this anymore)